lib/fastly.js

```diff
@@ -138,8 +128,24 @@ function handleFastlyResponse(successMessage) {
     const options = {};
     console.log(prettyjson.render(fastlyResponseBody, options));
     if (successMessage) {
-      console.log(chalk.bold.green(successMessage));
+      console.log(chalk.green(successMessage));
     }
     process.stdin.pause();
   };
 }
+
+function fastlyApiKeyPresent(program) {
+  if (!program.apikey) {
+    console.error(chalk.red(fastlyApiKeyError));
+    return false;
+  }
+  return true;
+}
+
+function fastlyServiceIdPresent(program) {
```

Comment:
I am unsure of which way to go here so I am going to leave it as is _for now_, but I did learn how to use the other options so that's a win.